File: Consigna1.py
import random
numeros = []
for i in range(10):
    n = random.randint(1, 100)
    print("Número generado", i+1, ":", n)
    numeros.append(n)
suma = sum(numeros)
promedio = suma / len(numeros)
print("La suma es:", suma)
print("El promedio es:", promedio)

File: Consigna1.cpp
#include <iostream>
#include <cstdlib>
#include <ctime>
using namespace std;

int main(){
    srand(time(0)); 
    int numeros[10];
    int suma = 0;
    for (int i = 0; i < 10; i++) {
        numeros[i] = rand() % 100 + 1; 
        cout << "Número generado " << i + 1 << ": " << numeros[i] << endl;
        suma += numeros[i];
    }
    cout << "La suma es: " << suma << endl;
    cout << "El promedio es: " << suma / 10.0 << endl;
    return 0;
}   